emmo/README.md

```diff
@@ -0,0 +1,114 @@
+
+5. `https://w3id.org/emmo/domain/{DOMAIN}/{PATH}/{MODULE} --> https://raw.githubusercontent.com/emmo-repo/{REPO_NAME}/master/{PATH}/{MODULE}.ttl`
+   - Target: `{PATH}/{MODULE}.ttl` file in master branch.
+
+6. `https://w3id.org/emmo/domain/{DOMAIN}/{VERSION}/{PATH}/{MODULE} --> https://raw.githubusercontent.com/emmo-repo/{REPO_NAME}/{VERSION}/{PATH}/{MODULE}.ttl`
+   - Target: `{PATH}/{MODULE}.ttl` file for given version and module.
+
+
+### Redirections to EMMO
+
+7. `https://w3id.org/emmo --> https://emmo-repo.github.io/EMMO/emmo{.html|.ttl}`
+   - If the user is accessing this from a browser, redirect to html documentation on GitHub Pages.
+   - Otherwise, redirect to the squashed `.ttl` file on GitHub Pages.
+   - Special case for inferred ontology: `https://w3id.org/emmo/emmo-inferred --> https://emmo-repo.github.io/EMMO/emmo-inferred.ttl`
+
+8. `https://w3id.org/emmo/ --> https://raw.githubusercontent.com/emmo-repo/EMMO/master/emmo.ttl`
+   - Alias: https://w3id.org/emmo/latest
+   - Alias: https://w3id.org/emmo/source
+   - Target: `emmo.ttl` file in the root of the master branch.
+
+9. `https://w3id.org/emmo/{VERSION} --> https://emmo-repo.github.io/EMMO/versions/{VERSION}/emmo{.html|.ttl}`
+   - If the user is accessing this from a browser, redirect to html documentation for given version on GitHub Pages.
+   - Otherwise, redirect to squashed `.ttl` file for given version on GitHub Pages.
+   - Special case for inferred ontology: `https://w3id.org/emmo/{VERSION}/emmo-inferred --> https://emmo-repo.github.io/EMMO/versions/{VERSION}/emmo-inferred.ttl`
+
+10. `https://w3id.org/emmo/{VERSION}/ --> https://raw.githubusercontent.com/emmo-repo/EMMO/{VERSION}/emmo.ttl`
+   - Alias: https://w3id.org/emmo/{VERSION}/source
+   - Target: `emmo.ttl` file in the root of branch/tag for the given version.
+
+11. `https://w3id.org/emmo/{PATH}/{MODULE} --> https://raw.githubusercontent.com/emmo-repo/EMMO/{PATH}/{MODULE}.ttl`
+   - Target: Turtle file for given EMMO module.
+
+12. `https://w3id.org/emmo/{VERSION}/{PATH}/{MODULE} --> https://raw.githubusercontent.com/emmo-repo/EMMO/{VERSION}/{PATH}/{MODULE}.ttl`
+   - Target: Turtle file for given version and module of EMMO.
+
+
+### Meaning of placeholders
+- `{NAME}`: Name part of an IRI, i.e., what follows after the hash sign.
+- `{VERSION}`: Version number. Must start with a digit to distinguish it from domain or path names.
+- `{PATH}`: Directory path within a github repository
+- `{MODULE}`: Filename of turtle file with the final `.ttl` stripped off
+- `{DOMAIN}`: Name of domain ontology. Initial `domain-` is stripped off.
+- `{REPO_NAME}`: Name of GitHub repository for domain ontology.
+
+
+## Expected organisation of repositories for domain ontologies
+
+### GitHub repository
+In the root of the repository, we should have a file called `{DOMAIN}.ttl`. This should be the main file for the ontology. It should exist to make the ontology FAIR.
+
+### GitHub Pages
```

Comment:
consider custom redirecting if needed: https://stackoverflow.com/questions/30167113/redirect-github-pages-to-custom-domain